### PR TITLE
13222 Check search result if MHR Number Search

### PR DESCRIPTION
**Pull request by ChasBoucher**
*Issue #:* /bcgov/13222

*Description of changes:*

1.  When user does an MHR Number Search only, there will be 1 result and it will be selected.. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/tables/mhr/SearchedResults.vue [hunk 1]
```diff
@@ -341,6 +341,11 @@ export default defineComponent({
         return result.includeLienInfo !== true ? { ...result, includeLienInfo: false } : result
       })
       localState.totalResultsLength = resp.totalResultsSize
+      if (localState.searchType === UISearchTypes.MHR_NUMBER && localState.totalResultsLength === 1) {
+        const val = true
+        // Select search result if an MHR Number Search and search results equals 1.
+        localState.results = localState.results.map(result => ({ ...result, selected: val }))
```
Comment (cameron-eyds) [inline ppr-ui/src/components/tables/mhr/SearchedResults.vue]: if `val` isn't use anywhere else in its scope, does it make sense to apply the value directly to the mapped property?
ie `selected: true`

**Review by cameron-eyds — APPROVED** [review]
Looks good just one minor comment 👍

File: ppr-ui/src/components/tables/mhr/SearchedResults.vue (hunk 1 above)
Comment (ChasBoucher) [reply ppr-ui/src/components/tables/mhr/SearchedResults.vue]: Good idea... I will make the change...

File: ppr-ui/src/components/tables/mhr/SearchedResults.vue [hunk 2]
```diff
@@ -341,6 +341,10 @@ export default defineComponent({
         return result.includeLienInfo !== true ? { ...result, includeLienInfo: false } : result
       })
       localState.totalResultsLength = resp.totalResultsSize
+      if (localState.searchType === UISearchTypes.MHR_NUMBER && localState.totalResultsLength === 1) {
+        // Select search result if an MHR Number Search and search results equals 1.
+        localState.results = localState.results.map(result => ({ ...result, selected: true }))
```
Comment (dimak1) [inline ppr-ui/src/components/tables/mhr/SearchedResults.vue:346]: If there is only one result, do we even need to use map() ?

**Review by dimak1 — APPROVED** [review]
Approved with one comment.

**Review by doug-lovett — APPROVED** [review]

File: ppr-ui/src/components/tables/mhr/SearchedResults.vue (hunk 2 above)
Comment (ChasBoucher) [reply ppr-ui/src/components/tables/mhr/SearchedResults.vue:346]: I did this to keep with existing logic... It may work but I have not tested..

**Pull request merged**